README.rst

```diff
@@ -235,7 +236,7 @@ Compatibility with GMT and Python/NumPy versions
 ======= ========== ========= =========
 PyGMT   GMT        Python    NumPy
 ======= ========== ========= =========
-0.4.x   >=6.2.0    >=3.7     >=1.17.0
+0.4.0   >=6.2.0    >=3.7     >=1.17.0
```

Comment:
Is the intention of changing 0.4.x to 0.4.0 here due to https://github.com/GenericMappingTools/pygmt/issues/1333#issuecomment-863524502? I.e. we want to pin PyGMT v0.4.1 to GMT 6.2.1?

Reply:
No, just was my interpretation of the checklist.

Reply:
Ah I see. I think we can revert this one line change as the PyGMT v0.4.x series should stick with these GMT/Python/NumPy versions (patch versions shouldn't update minimum Python/NumPy dependencies per [NEP29](https://numpy.org/neps/nep-0029-deprecation_policy.html#detailed-description)).

```suggestion
0.4.x   >=6.2.0    >=3.7     >=1.17.0
```